File: src/test/java/com/kevin/StudentTest.java
package com.kevin;

import com.kevin.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    void higestTest(){
        Student student = new Student("kevin",60,68);
        Assertions.assertEquals(68 , student.highest());
    }

    @Test
    void averageTest(){
        Student student = new Student("kevin", 60, 68);
        Assertions.assertEquals((60+68)/2 , student.getAverage());
    }
}

File: src/test/kotlin/com/kevin/StudentTest.kt
package com.kevin.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import java.lang.AssertionError

class StudentTest{
    @Test
    fun highestTest(){
        val student = Student("rose",50,89)
        Assertions.assertEquals(89,student.highest())
    }

    @Test
    fun bmiTest(){
        val student = Student("rose",50,89)
        Assertions.assertEquals((50+89)/2 , student.getAverage())
    }

    @Test
    fun gradingTest(){
        val student = Student("rose",50,89)
        Assertions.assertEquals('D',student.grading())
    }

}